### PR TITLE
Formatting of multiple jeps and index page

### PR DESCRIPTION
**Pull request by bitwiseman**

### CODE REVIEW

File: jep/209/README.adoc [hunk 1]
```diff
@@ -96,9 +96,12 @@ it’s in sync with the actual codebase, like Pipeline DSL documentation does.
 The configuration file should be validated without the need to actually run a Jenkins master. At same time we generate documentation we
 need to generate some schema that could also be used with adequate IDE / editor to assist writing configuration.
 
-
 == Motivation
 
+TODO: @ndeloof This need filling out.
```
[the remark this answers is not in the document]
Comment (jglick) [reply jep/209/README.adoc]: Well, best to do _both_ I suppose.

File: jep/200/README.adoc [hunk 2]
```diff
@@ -67,19 +67,10 @@ Java’s built-in API, from the Remoting layer;
 and XStream, for persisting configuration and settings.
 Both permit any Java class to be loaded merely by specifying its name in the input.
 
-For years, the Jenkins project has received reports of remote code execution (RCE) attacks involving these frameworks.
```
Comment (jglick) [inline jep/200/README.adoc]: Diff too long to review; assuming you just rearranged content here.

File: jep/report-jep-conformance [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+REL_SCRIPT_DIR="`dirname \"$0\"`"
+SCRIPT_DIR="`( cd \"$REL_SCRIPT_DIR\" && pwd )`"
+PROJECT_ROOT="`( cd \"$SCRIPT_DIR/..\" && pwd )`"
+
+# `report-jep-conformance <jep-number>`
```
Comment (jglick) [inline jep/report-jep-conformance]: Make this part of a `Jenkinsfile` so submitters do not forget?

Comment (bitwiseman) [reply jep/report-jep-conformance]: @jglick This repository does not have that yet.

File: jep/200/README.adoc (hunk 2 above)
Comment (bitwiseman) [reply jep/200/README.adoc]: @jglick  Yes.

File: jep/4/README.adoc [hunk 4]
```diff
@@ -72,7 +72,6 @@ to tackle specific areas of focus within the project. This proposal describes a
 become more formalized, with consistent expectations of their operation, to
 help new contributors join areas of effort which align with their interests.
 
-[[specification]]
```
[the remark this answers is not in the document]
Comment (bitwiseman) [reply jep/4/README.adoc]: Not needed.  The section name produces this label automatically.

File: jep/209/README.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bitwiseman) [reply jep/209/README.adoc]: @oleg-nenashev Sure.

**Pull request merged**